content/en/docs/instrumentation/cpp/_index.md

```diff
@@ -8,22 +8,7 @@ description: >
   A language-specific implementation of OpenTelemetry in C++.
 ---
 
-This is the OpenTelemetry for C++ documentation. OpenTelemetry is an
```

Comment:
Is there a way to do a docs comment or something that won't render? Since language SIG owners are already used to going into these pages to update status, we should leave a comment here asking them to update it in the yaml file.

Reply:
Good point, I added a small comment to each page, we probably can/should do the same throughout the docs